utils/error.js

```diff
@@ -0,0 +1,16 @@
+const catchAsync = (func) => {
+  return (req, res, next) => {
+    func(req, res, next).catch(next);
```

Comment:
```suggestion
    func(req, res).catch(next);
```
더 명확하게는 func(req, res, next)에서 next는 없어도 됩니다! 감싸주는 함수가 (req, res) 두가지 인자만 취급하고 있기 때문입니다.